README.md

```diff
@@ -12,20 +12,21 @@ The content of the generated file should be added to the appropriate `client_sec
 
 Building
 --------
+Windows users must ensure that the following value is set in Docker Engine > Edit JSON: “experimental: true”.
 
-When building the amps (`mvn clean package`) you should also include a combination of the following profiles (`-P`). If not include, the defaults are applied.
+When building the amps (`mvn clean package`) you should also include a combination of the following profiles (`-P`). If not included, the defaults are applied.
 
 **Platform**
```

Comment:
This heading doesn't make sense with new changes, possibly change to "Docker images" or "Building development images" or "Docker images for testing"

Reply:
good point